tests/input/test_custom_coordinate_release.yml

```diff
@@ -0,0 +1,26 @@
+parcel:
+  numpy1: 1.0.0
+  numpy2: 2.0.0
+letter:
+  py38:
+    numpy1: 1.38.0
+    numpy2: 2.28.0
+  py311:
+    numpy1: 1.311.0
+    numpy2: 2.211.0
+box:
+  rhel8:
+    py38:
+      numpy1: 1.38.8
+      numpy2: 2.28.8
+    py311:
+      numpy1: 1.311.8
+      numpy2: 2.211.8
+  rhel9:
+    py38:
```

Comment:
Is `numpy1` possible at the second level here also, or only at the third level. Since rhel and py works at both levels interchangibly, it is natural to think that this also applies to the third component.

Reply:
This might be a misconception. I think you can have (in `main`) python-tags, or rhel-tags with python-tags inside.
How we want the end result is obviously up for debate.

Reply:
Restricting it to only the third level is ok for now!

Reply:
I've created an issue that describes an possible way of extending the komodo transpiling functionality.
https://github.com/equinor/komodo/issues/526